packages/calcite-components/src/components/pagination/pagination.scss

```diff
@@ -5,6 +5,7 @@
 
 .list {
   @apply flex list-none m-0 p-0;
+  column-gap: 2px;
```

Comment:
Should these be using spacing tokens (`--calcite-spacing-base`)?

Reply:
Thanks for catching that! I updated this to use `--calcite-spacing-base`.